navapbc/template-application-nextjs
# Apply security updates

## Ticket

N/A

## Changes
> What was added, updated, or removed in this PR.

- Updates:
    - next-i18next from ^11.0.0 to ^11.2.2
    - @types/jest from ^27.5.1 to ^28.1.6
    - @types/node from ^17.0.34 to ^18.0.6
    - @typescript-eslint/eslint-plugin from ^5.25.0 to ^5.26.0
    - @typescript-eslint/eslint-plugin from ^5.30.7 to ^5.30.7
    - eslint from ^8.15.0 to ^8.20.0
    - eslint-plugin-storybook from ^0.5.12 to ^0.6.1
    - ts-jest from ^28.0.2 to ^28.0.7

## Context for reviewers
> Testing instructions, background context, more in-depth details of the implementation, and anything else you'd like to call out or ask reviewers. Explain how the changes were verified.

Apply outstanding security updates flagged by dependabot.

## Testing
> Screenshots, GIF demos, code examples or output to help show the changes working as expected. ProTip: you can drag and drop or paste images into this textbox.

1. `docker-compose up -d --build`
2. Check `localhost:3000` to ensure the site still functions as expected
3. `docker-compose exec nextjs yarn storybook`
4. Check `localhost:6006` to ensure storybook still functions as expected
5. (Optional) Run `yarn format && yarn lint && yarn ts:check && yarn build && yarn test` or let Github CI handle that
<img width="802" alt="Screen Shot 2022-07-19 at 1 39 18 PM" src="https://user-images.githubusercontent.com/67701/179844279-4744f30f-e10f-4e7c-bb52-7723d286cece.png">
<img width="994" alt="Screen Shot 2022-07-19 at 1 39 31 PM" src="https://user-images.githubusercontent.com/67701/179844282-191f526c-511d-4015-a620-a1bc78fabf21.png">
